pymeasure/instruments/advantest/advantestR624X.py

```diff
@@ -0,0 +1,1040 @@
+
+        If the output data format is specified as binary format,
+        the terminator is fixed to <EOI> only and the terminator selection is ignored.
+
+        delimiter_format
+        1 ASCII format with header
+        2 No header, ASCII format
+        3 Binary format
+
+        1st character header:
+            A: Normal measurement data
+            B: Measurement data during overrange
+            C: Compliance (limiter) is working.
+            D: Oscillation detection is working.
+            E: [Indicates the generated data]
+            F: Measurement data when an error occurs in the search measurement
+            Z: Measurement data is not stored in the buffer memory.
+
+        2nd character header:
+            A: A-channel data during asynchronous operation (A-channel generation data)
+            B: B-channel data during asynchronous operation (B channel generation data)
+            I: A-channel data for synchronous, sweeping, delayed sweep, and double synchronous sweep operations.
+            J: B-channel data for synchronous, sweeping, delayed sweep, and double synchronous sweep operations.
+
+        3rd character header:
+            A: Current generation, voltage measurement (ISVM) [Current generation]
+            B: Voltage generation, current measurement (VSIM) [Voltage generation]
+            C: Current generation, current measurement (ISIM)
+            D: Voltage generation, voltage measurement (VSVM)
+            E: Current generation, external voltage measurement (IS, EXT, VM)
+            F: Voltage generation, external current measurement (VS, EXT, IM)
+            G: Current generation, external current measurement (IS, EXT. IM)
+            H: Voltage generation, external voltage measurement (VS, EXT, VM)
+            Z: The measurement data is not stored in the buffer memory.
+
+        4th character header:
+            A: No operation (fixed to A)
+            B: Null operation result
+            C: The result of the comparison operation is GO.
+            D: The result of the comparison operation is LO.
+            E: The result of the comparison operation is HI.
+            F: The result of null operation + comparison operation is GO.
+            G: The result of null operation + comparison operation is LO.
+            H: The result of null operation + comparison operation is HI.
+            Z: Measurement data is not stored in the buffer memory.
+
+        block_delimiter
+        1 Make it the same as the terminator.
+        2 Use semicolon ;
+        3 Use comma ,
+
+        terminator
+        1 CR, LF<EOI>
+        2 LF<EOI>
+        3 LF
+        4 <EOI>
+        """
+        self.write(f'fmt 0, {delimiter_format}, {block_delimiter}, {terminator}')
+        self.check_errors()
+
+    service_request_enable_register = Instrument.control(
+        '*sre?', '*sre %i',
+        """Property controlling the contents of the service request enable register (SRER).
```

Comment:
According to Python standards, a docstring should be in imperative form (Do something).

Additionally, we at pymeasure want to start `control` (and derivatives) docstrings with "Control" (or "Measure" or "Set", for `measurement` and `setting`), see [documentation](https://pymeasure.readthedocs.io/en/latest/dev/coding_standards.html).
For examples look through [adding instruments](https://pymeasure.readthedocs.io/en/latest/dev/adding_instruments.html).

This change is quite recent. I guess you orientated yourself at older examples, when you wrote the instrument driver.

Reply:
Did a complete overhaul of the documentation, spend quite a lot of time on this. Would love to hear your feedback.

Reply:
Sorry forgot about 'Control' and 'Set', I'll change that

Reply:
In my last commit I changed "Set" and "Control" on all .setting and .control functions. My .measurement functions don't measure anything, as most of them are registers for example. Should I change these to regular returning .ask() functions?

Reply:
I hope to get to a review tomorrow.

You may use "measurement", even though it is just a setting (for example mode of the device).
At first it seems strange, but in the end you "Measure" the device configuration.

Reply:
Ok, resolved